src/HttpRequestParser.cpp

```diff
@@ -90,6 +91,12 @@ void HttpRequestParser::parseRequestLine(std::string &requestLine, HttpRequest *
 	validateVersion(requestLine);
 }
 
+void HttpRequestParser::validateSize(HttpRequest *request, Server *server)
+{
+	if (request->getMethod() == Util::METHOD::POST && request->getContentLength() > server->getClientMaxBodySize())
```

Comment:
Also CGI_POST content length should be checked!

Reply:
done